tests/disaster-recovery/regional-dr/test_failover_and_relocate.py

```diff
@@ -201,6 +241,44 @@ def test_failover_and_relocate(
         logger.info(f"Waiting for {wait_time} minutes to run IOs")
         sleep(wait_time * 60)
 
+        # Get lastGroupSyncTime after failover
+        for drpc_data in TimeoutSampler(60, 3, drpc_obj.get):
+            post_failover_last_group_sync_time = drpc_data.get("status").get(
+                "lastGroupSyncTime"
+            )
+            if post_failover_last_group_sync_time:
+                logger.info("After failover - Obtained lastGroupSyncTime.")
+                if post_failover_last_group_sync_time != last_group_sync_time:
+                    logger.info(
+                        "After failover - Verified: lastGroupSyncTime after is different from initial value."
+                    )
+                    break
+            logger.info(
+                "lastGroupSyncTime is not available after failover or it is equal to the previous value. Retrying."
```

Comment:
The value is reset the moment we failover, so the log here is incorrect. It would either be blank or have some random value with @sidhant-agrawal can confirm.

Reply:
Updated the logs.